### ISSUE TITLE
[Anti-adblock] hiddenobjectgames.com

### ISSUE THREAD

**Issue by jspenguin2017** [uBlockOrigin/uAssets#2889]
### URL(s) where the issue occurs

`https://www.hiddenobjectgames.com/game/Playground+Differences`

### Describe the issue

There's a timer that delays loading if adblocker is detected

### Screenshot(s)

![image](https://user-images.githubusercontent.com/7283682/42725653-84e8e70e-8744-11e8-8149-2c3c2b6859bf.png)

### Versions

- Browser/version: Chrome 69
- uBlock Origin version: 1.16.12

### Settings

English default.

**Comment by zaphodbeebblebrox** [comment]
For nano you might want to inject your dummy ima3.

**Comment by jspenguin2017** [comment]
I used to serve the surrogate for every website, but quite a few websites break because of it. 

Some websites have fallback code when IMA SDK is blocked and somehow the fallback code is less buggy than the regular code. It's a mess and I don't really want to spend more time debugging it, so I disabled my surrogate for now. 

This is my current implementation: https://github.com/jspenguin2017/uBlockProtector/blob/master/src/resources/ima3.js
